account_financial_risk/models/res_company.py

```diff
@@ -9,7 +9,7 @@ class ResCompany(models.Model):
 
     invoice_unpaid_margin = fields.Integer(
         string="Maturity Margin",
-        help="Days after due date to set an invoice as unpaid."
-             "The change of this field recompute all partners risk,"
+        help="Days after due date to set an invoice as unpaid. "
```

Comment:
If you change this text, then you should update the msgid in all the PO/POT files.